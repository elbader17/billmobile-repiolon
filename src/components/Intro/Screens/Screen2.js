import React, { Component } from 'react';
import { Text, View } from 'react-native';
import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
import style from './style';

class Screen2 extends Component {
  render() {
    return (
      <KeyboardAwareScrollView>
        <View style={style.silde}>
          <Text style={style.number}>2</Text>
          <Text style={style.textTittle}>Hacemos práctica la facturación</Text>
          <Text style={style.textDescription}>No tendrás que cargar la misma información 
          dos veces. Nuestro sistema guarda y ayuda a hacer más ágil tu facturación.</Text>
        </View>
      </KeyboardAwareScrollView>
    );
  }
}

export default Screen2;