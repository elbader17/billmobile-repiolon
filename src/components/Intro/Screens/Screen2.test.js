import React from 'react';
import renderer from 'react-test-renderer';
import Screen2 from './Screen2';

describe('Rendering Component Screen2', () => {
  test('renders correctly', () => {
    const component = renderer.create(<Screen2 />).toJSON();
    expect(component).toMatchSnapshot();
  });
})
