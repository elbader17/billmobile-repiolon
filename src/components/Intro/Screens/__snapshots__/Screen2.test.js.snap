// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering Component Screen2 renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#3687d1",
      "height": 1334,
      "justifyContent": "center",
      "width": 750,
    }
  }
>
  <Text
    style={
      Object {
        "bottom": 422,
        "color": "#fff",
        "fontFamily": "Lato-Regular",
        "fontSize": 80,
        "lineHeight": 90,
        "position": "absolute",
      }
    }
  >
    2
  </Text>
  <Text
    style={
      Object {
        "alignItems": "center",
        "color": "#fff",
        "fontFamily": "Lato-Semibold",
        "fontSize": 18,
        "height": 33,
        "lineHeight": 32,
        "position": "absolute",
        "textAlign": "center",
        "top": 287,
        "width": 289,
      }
    }
  >
    Hacemos práctica la facturación
  </Text>
  <Text
    style={
      Object {
        "alignItems": "center",
        "color": "#fff",
        "fontFamily": "Lato-Regular",
        "fontSize": 14,
        "height": 66,
        "lineHeight": 21,
        "position": "absolute",
        "textAlign": "center",
        "top": 332,
        "width": 289,
      }
    }
  >
    No tendrás que cargar la misma información dos veces. Nuestro sistema guarda y ayuda a hacer más ágil tu facturación.
  </Text>
</View>
`;
